parentheses/algorithm.go

```diff
@@ -0,0 +1,27 @@
+// Package implements function BalancedString, that verifies if the given string is a balanced sequence of brackets
+package parentheses
+
+// BalancedString verifies if the given string is a balanced sequence of brackets
+var brackets = map[string]string{"{": "}", "(": ")", "[": "]"}
+
+func BalancedString(s string) bool {
+	stack := make([]string, 0, 16)
```

Comment:
why is cap =  16?

Reply:
> 
What the capacity should be?

Reply:
don't use magic number that don't have a reason to be there. If you don't have an explanation for 16, stick with the default.
Here you should base your `cap` on the size of `s`

Reply:
Done